schema.ts

```diff
@@ -1186,7 +1186,7 @@ function checkSchemaPropertyIsRequired<K extends string, L extends string, T ext
 ) {
   const {properties, codaType} = schema;
   assertCondition(
-    properties[field].required,
+    properties[field]?.required,
```

Comment:
Should we add a new check right above it that `properties[field]` is actually defined? Seems easy enough, to be able to accurate stately what the issue is in that case.

Reply:
Updated the error to `Error: displayProperty set to undefined field "Name"`

One unfortunate thing is that it'll say `displayProperty` or `idProperty` even if the user was using the deprecated `primary` or `id` parameters, but my first attempt to distinguish between those cases didn't work and I think it's probably not worth spending more time on. Eventually `id` and `primary` will be unsupported anyway

Reply:
Yeah I don't think it's worth spending more time on. Onward!